src/server/pbsd_main.c

```diff
@@ -1074,6 +1074,11 @@ main(int argc, char **argv)
 		pbs_scheduler_addr = get_hostaddr(pbs_conf.pbs_primary);
 	}
 
+	/* set the scheduler name if it is set in an env var or conf file */
+	if (pbs_conf.pbs_scheduler_host_name != NULL) {
+		pbs_scheduler_addr = get_hostaddr(pbs_conf.pbs_scheduler_host_name);
+	}
+
```

Comment:
I'm not sure how this should interact with failover.  The non-failover case is obvious.  The comment above the failover case says the default scheduler is on the primary's host.  If we change it to this, what happens when the secondary takes over?  Is it still here?  Also, how does this interact with multi-sched?

Reply:
If this is set, then I think it should be what is used (over-written only by -S), this is how the other /etc/pbs.conf file variables work.  That is why the code is located after the failover code. Also, for the secondary server that might be located on a different host, it will have it's own configuration file, and should have the same PBS_SCHEDULER_HOST_NAME set.  What do you think?
For multisched it should not matter.  It's a setting used for the server. But just in case I'll try running a few tests to see.

Reply:
That's a good point that the secondary will have its own conf file.  As for multi-sched, right now they all have to be on the server's host.  It was always in the plans to make it so they could be elsewhere.  I don't know if that changes things.